build/import/Packages.targets

```diff
@@ -126,8 +126,11 @@
     <PackageReference Update="Roslyn.Diagnostics.Analyzers"                                           Version="3.0.0-beta2.20059.3+77df2220" />
 
     <!-- NuGet -->
-    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="5.6.0-preview.2.6505" />
-    <PackageReference Update="NuGet.VisualStudio"                                                     Version="5.6.0-preview.2.6505" />
+    <!-- TODO undo version rollback (was needed as VS ships this lower version) -->
+    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="5.6.0-preview.2.6489" />
+    <PackageReference Update="NuGet.VisualStudio"                                                     Version="5.6.0-preview.2.6489" />
+    <PackageReference Update="NuGet.ProjectModel"                                                     Version="5.6.0-preview.2.6489" />
+    <PackageReference Update="NuGet.Common"                                                           Version="5.6.0-preview.2.6489" />
```

Comment:
@davkean you upgraded this recently. Do you see a problem with this slight downgrade?

Reply:
Does it build? If so, this is fine.